src/components/ComponentPages/DirectSupportedCamps/__test__/directSupportedCamps.test.tsx

```diff
@@ -195,7 +195,7 @@ describe("Direct Support camps page", () => {
       />
     );
     expect(screen.getByText("Remove")).toBeTruthy();
-    expect(screen.getByText("Remove Support")).toBeInTheDocument();
+    expect(screen.getAllByText("Remove Support")).toBeTruthy();
```

Comment:
Please move "Remove Support" to common message or label file

Reply:
Done